client/src/components/records.tsx

```diff
@@ -1,52 +1,77 @@
+                        display: resource.item[0]?.productOrService?.coding[0]?.display || 'Unknown Medication',
+                        amount: resource.item[0]?.adjudication[7]?.amount?.value
+                    }
+                });
+                setRecords(records);
+            });
+    }, [])
 
-export default function Records({ }) {   
-    //TODO: Fetch real data 
-    const items = [
-        {
-            id: 1,
-            code: '0',
-            display: 'Lab tests - other (Medicare fee schedule)',
-            value: '$125.98'
-        },
-        {
-            id: 2,
-            code: '45385',
-            display: 'Local carrier non-durable medical equipment, prosthetics, orthotics, and supplies (DMEPOS) claim',
-            value: '$350'
-        },
-        {
-            id: 3,
-            code: 'P8D',
-            display: 'Endoscopy - colonoscopy',
-            value: '$1499.99'
-        }
-    ]    
     return (
         <div className='full-width-card'>
             <Table className="ds-u-margin-top--2" stackable stackableBreakpoint="md">
                 <TableCaption>Medicare claims data</TableCaption>
                 <TableHead>
                     <TableRow>
-                    <TableCell id="column_1">Code</TableCell>
-                    <TableCell id="column_2">Name</TableCell>
-                    <TableCell id="column_3">Price</TableCell>
+                        <TableCell id="column_1">Code</TableCell>
+                        <TableCell id="column_2">Name</TableCell>
+                        <TableCell id="column_3">Price</TableCell>
                     </TableRow>
                 </TableHead>
                 <TableBody>
-                    {items.map(item=>(
-                        <TableRow key={item.id}>
-                            <TableCell stackedTitle="Document title" headers="column_1">
-                                {item.code}
-                            </TableCell>
-                            <TableCell stackedTitle="Description" headers="column_2">
-                                {item.display}
-                            </TableCell>
-                            <TableCell stackedTitle="Year" headers="column_3">
-                                {item.value}
-                            </TableCell>
-                        </TableRow>
-                    ))}
+
+                    {records.map(record => {
+                        return (
+                            <TableRow key={record.id}>
+                                <TableCell stackedTitle="Document title" headers="column_1">
+                                    {record.code}
+                                </TableCell>
+                                <TableCell stackedTitle="Description" headers="column_2">
+                                    {record.display}
+                                </TableCell>
+                                <TableCell stackedTitle="Year" headers="column_3">
```

Comment:
```suggestion
                                <TableCell stackedTitle="Price" headers="column_3">
```

Reply:
Not sure where these "stackedTitle" are coming from... but lets use more realistic names

Reply:
For sure!  I totally missed these.  Thanks for catching this!  Done 'Cost'